src/Enum.js

```diff
@@ -10,6 +10,7 @@ export var SUPPORTS = '@supports'
 export var DOCUMENT = '@document'
 export var KEYFRAMES = '@keyframes'
 export var PAGE = '@page'
+export var FONT_FACE = '@font-face'
 export var FONT_FEATURE_VALUES = '@font-feature-values'
```

Comment:
```diff
-export var FONT_FACE = '@font-face'
```

Reply:
I have actually left this here deliberately. IMHO good to have all possible values listed in a file like this - even if some stay unused in the code (rollup or any other minifier will just remove this anyway).

Let me know if you want to keep this or remove.

Reply:
I don't even think we use any of the at-rule constants there. 

But i see your point, so maybe let's add the other at-rules constants like `@viewport, @counter-style` etc.

Reply:
Let's remove at-moz-document, since it's not official. We can remake it with MOZ + DOCUMENT if need me.

Reply:
Ok, I've removed it from this listing - but I still keep the test and support in the code. Or should I remove those as well?

Reply:
Everything else is ok. The code works on the presence of the hyphen, i think this would make it generic for all similar vendor specific at-rules.

Reply:
Ok, merging this in then!